customize-sh-reporter/readme.md

```diff
@@ -0,0 +1,6 @@
+---
+
+We are using this image as our base image and to customize it for our own use we add fews  file 
+customTemplate.tex and logo.png.
+
+In case we make any change ,still we need to keep customTemplate.tex & logo.png in dockerfile as it require for custmozination.
```

Comment:
```suggestion
---
# Customized Build - Grafana Reporter

![](https://camo.githubusercontent.com/64f8905651212a80869afbecbf0a9c52a5d1e70beab750dea40a994fa9a9f3c6/68747470733a2f2f617765736f6d652e72652f62616467652e737667)

## Final Docker Image

We are going to build a final image on top of reporter base image.
(shpvt01.azurecr.io/shinternal/infer/grafana-pdf-reporter:2.3.1 or izakmarais/grafana-reporter)

This step is required because of following both files are used inside pdf reporter. 

- customTemplate.tex
- logo.png


> Note: When ready, simply use the Dockerfile to build the final image.


## License

MIT

**Happy Coding**

```